use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -97,24 +96,44 @@ chipset by executing the following.
 $ for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
 ```
 
-### Install necessary bundles for Clear Linux and disable kernel updates
+### Install necessary bundles for Clear Linux
 
-You need the Linux kernel headers to compile the out-of-tree kernel modules to
-support Intel QAT on the host. The `os-clr-on-clr` is a large development 
-bundle that you can install instead to include everything below with 
-additional items. This guide installs the Linux LTS kernel version `4.19` from
-2018 because the crypto modules are set to `m` and not `y`, which allows you
-to replace them with the out-of-tree driver. Otherwise you need to build your
-own Clear Linux kernel with the QAT drivers compiled in.
+Clear Linux version 30780 (Released August 13, 2019) includes a 
+`linux-firmware-qat` bundle that has the necessary QAT firmware along with a
+functional QAT host driver that can work with Kata Containers. Any version of
+Clear Linux after 30780 should work without having to replace the existing QAT
+host drivers. Other Linux distributions and previous versions of Clear Linux
+will require compiling and installing the QAT driver to the host. Those optional
+instructions are further below. In addition, the `os-clr-on-clr` is a large 
+development bundle that you can install instead to include everything below
+with additional items.  
 
 ```sh
 $ sudo swupd autoupdate --disable
-$ sudo swupd bundle-add network-basic kernel-lts2018 linux-lts-dev linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
+$ sudo swupd bundle-add network-basic linux-firmware-qat linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
```

Comment:
check the list for what is really needed (for building kata rootfs/kernel + QAT)

Reply:
I did that a couple months ago. This was the smallest bundle set I could get working. If any of these were missing I couldn't compile the QAT driver or couldn't do something else.  Given that Clear Linux now has the firmware bundle I don't need to compile against the host kernel so I could remove the linux-dev package.

Reply:
OK. 

Drop `sudo swupd autoupdate --disable`

Reply:
Will do.